linkerd/app/inbound/src/policy.rs

```diff
@@ -133,6 +134,19 @@ impl AllowPolicy {
         ServerLabel(self.server.borrow().meta.clone())
     }
 
+    pub fn ratelimit_label(&self, error: &RateLimitError) -> HTTPLocalRateLimitLabels {
+        let error = match error {
+            RateLimitError::Total(rps) => format!("total({})", rps),
+            RateLimitError::PerIdentity(rps) => format!("per-identity({})", rps),
+            RateLimitError::Override(rps) => format!("override({})", rps),
```

Comment:
We should not use label values to encode the RPS like this. Let's keep label values simple, encoding only one piece of information.

What do you think about making this a simple `scope="total|identity"` -- the "per-identity" limit is really the default when there is no override for a client.

Reply:
Sounds good to me. Changed in 6fc6d756.